components/authz-service/server/v2/authz.go

```diff
@@ -60,9 +66,18 @@ func (s *authzServer) ProjectsAuthorized(
 	if err != nil {
```

Comment:
The very first thing in this function (not shown) is this:

projectsAuthorized, err := s.engine.V2ProjectsAuthorized(ctx,
		engine.Subjects(req.Subjects),
		engine.Action(req.Action),
		engine.Resource(req.Resource),
		engine.ProjectList(req.ProjectsFilter...))

But remember the above is being used for both v2.0 and v2.1. If it is v2.0, then you should pass `AllProjectsID` instead of `req.ProjectsFilter` as the last param above. That way, if the system is on v2.0 and the user does include project headers, it will ignore them and ask for all projects in the above call.